.github/scripts/generate-prdoc.py

```diff
@@ -117,7 +133,7 @@ def setup_parser(parser=None, pr_required=True):
 		parser = argparse.ArgumentParser()
 	parser.add_argument("--pr", type=int, required=pr_required, help="The PR number to generate the PrDoc for.")
 	parser.add_argument("--audience", type=str, nargs='*', choices=allowed_audiences, default=["todo"], help="The audience of whom the changes may concern. Example: --audience runtime_dev node_dev")
```

Comment:
nit: may be worth moving the audience map into separate var in this scope, so the allowed_audiences could be formed dynamically out of aliases keys?